parse/index.js

```diff
@@ -381,6 +386,49 @@ export default ({
     }
   }
 
+  function parseViewSetFlowToBasedOnData(block) {
+    let dataProp = block.properties.find((item) => item.name === 'data')
+    if (dataProp) {
```

Comment:
since there's nothing beyond the if, should we do it like `if (!dataProp) return` and continue the rest of the flow at the same level?